src/test/java/hudson/plugins/active_directory/docker/PlanetExpressTest.java

```diff
@@ -0,0 +1,63 @@
+package hudson.plugins.active_directory.docker;
+
+import org.jenkinsci.test.acceptance.docker.DockerContainer;
+import org.jenkinsci.test.acceptance.docker.DockerFixture;
+import org.jenkinsci.test.acceptance.docker.DockerRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+
+
+/**
+ * Tests the plugin when logging in to rroemhild/test-openldap
+ */
+public class PlanetExpressTest   {
+
+    @Rule
+    public DockerRule<PlanetExpress> docker = new DockerRule<PlanetExpress>(PlanetExpress.class);
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    @LocalData
```

Comment:
I doubt you can use `@LocalData` in this context, since you need to wait for the container to be fully up and running before you try to use it. Just call `setSecurityRealm` programmatically instead.